<!-- Line Follower launch file -->

<launch>
	<group ns="MR_LineFollower">
		<node pkg="mr_line_follower" name="LineFollower" type="mr_line_follower">
			<param name="imageInput_sub" value="/mrCamera/image"/> 
			<param name="imageFiltered_pub" value="/mrLineFollower/image_filtered"/> 
		</node>
	</group>
</launch>
